Source/dom/models/Document.js

```diff
@@ -134,6 +134,18 @@ export class Document extends WrappedObject {
     return toArray(documentData.allSymbols()).map(wrapObject)
   }
 
+  getSharedLayerStyles() {
+    const documentData = this._getMSDocumentData()
+    return toArray(documentData.layerStyles().sharedStyles()).map(wrapObject)
```

Comment:
This will only return styles local to the document. There's also an `allLayerStyles` (and equivalent `allTextStyles`) method which will include any foreign styles used within the document. This is maybe be design, but I thought I'd point it out just in case.